src/client/test-run/iframe.js.mustache

```diff
@@ -50,10 +50,10 @@
             return;
         }
 
-        function runInContext (stepName, step, stepNum, runner) {
+        function runInContext (stepName, step, runner) {
             runner([stepName], [function () {
                 eval('(' + step + ').call(this)');
-            }], stepNum, true);
+            }]);
```

Comment:
as i see the last argument is missed
`-            }], stepNum, true);`
`+            }]);`

We should delete just `stepNum` but not `true`